experiments/dht/dht_module.py

```diff
@@ -17,16 +21,45 @@ def __init__(self, experiment):
         super(DHTModule, self).__init__(experiment, DHTDiscoveryCommunity)
         self.start_time = 0
 
+        self.callbacks = {}
+
     def on_id_received(self):
         super(DHTModule, self).on_id_received()
         self.tribler_config.set_dht_enabled(True)
 
+        with open('autoplot.txt', 'a') as output_file:
```

Comment:
Have a look at https://github.com/Tribler/gumby/blob/e8c77759bc6cfbffb75a272e92c98ed4ced1eb4f/experiments/gigachannel/gigachannel_module.py

There is no need to make the autoplot directories yourself.